corehq/apps/userreports/expressions/specs.py

```diff
@@ -314,6 +315,60 @@ def _get_forms(self, case_id, context):
         return xforms
 
 
+class FormsInDateExpressionSpec(JsonObject):
+    type = TypeProperty('get_case_forms_in_date')
```

Comment:
Actually decided I don't like this being in the main ucr app since it relies on ES. going to put it in the custom icds folder